packages/flame/test/components/position_component_test.dart

```diff
@@ -717,6 +717,96 @@ void main() {
         expect(child.position, Vector2(3, 2));
         expect(child.absolutePosition, Vector2(15, 21));
       });
+
+      test('lookAt', () {
+        final comp = PositionComponent();
+
+        final targets = [
+          Vector2(0, 1),
+          Vector2.all(2),
+          Vector2(-1, 0),
+          Vector2.all(-50)
+        ];
+        final expectedAngles = [pi, (3 * pi / 4), (-pi / 2), (-pi / 4)];
+
+        for (var i = 0; i < targets.length; ++i) {
+          final target = targets.elementAt(i);
+          final angle = expectedAngles.elementAt(i);
+
+          expectDouble(comp.angleTo(target), angle, epsilon: 1e-10);
+
+          comp.lookAt(target);
+          expectDouble(comp.angle, angle, epsilon: 1e-10);
+        }
+      });
+
+      test('lookAt with native angle', () {
+        final comp = PositionComponent(nativeAngle: pi / 2);
+
+        final targets = [
+          Vector2(0, 1),
+          Vector2.all(2),
+          Vector2(-1, 0),
+          Vector2.all(-50)
+        ];
+        final expectedAngles = [pi / 2, (pi / 4), -pi, (-3 * pi / 4)];
+
+        for (var i = 0; i < targets.length; ++i) {
+          final target = targets.elementAt(i);
+          final angle = expectedAngles.elementAt(i);
+
+          expectDouble(comp.angleTo(target), angle, epsilon: 1e-10);
+
+          comp.lookAt(target);
+          expectDouble(comp.angle, angle, epsilon: 1e-10);
+        }
+      });
+
+      test('lookAt with nested components', () {
+        late PositionComponent comp;
+
+        PositionComponent(
+          angle: pi / 2,
+          children: [
+            PositionComponent(
+              angle: pi / 2,
+              children: [
+                comp = PositionComponent(
+                  nativeAngle: -pi,
+                )
+              ],
+            )
+          ],
+        );
+
+        final targets = [
+          Vector2(0, 1),
+          Vector2.all(2),
+          Vector2(-1, 0),
+          Vector2.all(-50)
+        ];
+        final expectedAngles = [pi, (3 * pi / 4), -pi / 2, (-pi / 4)];
+
+        for (var i = 0; i < targets.length; ++i) {
+          final target = targets.elementAt(i);
+          final angle = expectedAngles.elementAt(i);
+
+          expectDouble(comp.angleTo(target), angle, epsilon: 1e-10);
+
+          comp.lookAt(target);
+          expectDouble(comp.angle, angle, epsilon: 1e-10);
+        }
+      });
+
+      test('lookAt corner cases', () {
+        final comp = PositionComponent(position: Vector2(-20, 50));
```

Comment:
No abbreviations, comp -> component

Reply:
Fixed this for mine as well as existing tests too.